cli/azd/extensions/registry.json

```diff
@@ -1,4 +1,78 @@
 {
-    "$schema": "registry.schema.json",
-    "extensions": []
+  "extensions": [
+    {
+      "id": "microsoft.azd.demo",
+      "namespace": "demo",
+      "displayName": "Demo Extension",
+      "description": "This extension provides examples of the AZD extension framework.",
+      "versions": [
+        {
+          "version": "0.0.2",
```

Comment:
Just wondering here if you have ideas around when a version could be removed, since it may be a maintenance burden for us.

I am familiar with krew for kubectl, which only registers the latest version since extensions are meant to be slightly more volatile, and the strategy they advise users is "always use latest". Example [here](https://github.com/kubernetes-sigs/krew-index/blob/master/plugins/browse-pvc.yaml)

Reply:
Another great question.  We'll need to have some ability to deprecate old versions that we don't want users to install.  Lets discuss and we can figure something out later.